client/kafka/v2/kafka_test.go

```diff
@@ -51,3 +53,15 @@ func TestBuilder_CreateAsync(t *testing.T) {
 		})
 	}
 }
+
+func Test_DefaultConsumerSaramaConfig(t *testing.T) {
+	sc, err := DefaultConsumerSaramaConfig("name", false)
+	assert.NoError(t, err)
+	assert.True(t, strings.HasSuffix(sc.ClientID, fmt.Sprintf("-%s", "name")))
```

Comment:
Should we not test all the settings and not only client id?

Reply:
This test is copied from here: https://github.com/beatlabs/patron/pull/409/files#diff-93799b4fcacca1ef1ca0b3b8912b456c6708b216a7f2d00f33ea2440c93b6dddL44

It was already testing only the ClientID; I will add a test for the isolation level.